automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/security/AuthDomainSpec.scala

```diff
@@ -83,9 +83,7 @@ class AuthDomainSpec extends FreeSpec /*with ParallelTestExecution*/ with Matche
                 val cloneModal = summaryPage.clickCloneButton()
                 cloneModal.readLockedAuthDomainGroups() should contain(authDomainName)
 
-                register cleanUp {
-                  api.workspaces.delete(projectName, cloneWorkspaceName)(authToken)
-                }
+                register cleanUp api.workspaces.delete(projectName, cloneWorkspaceName)(authToken)
```

Comment:
(optional) because `authToken` is an implicit (line 75), you should just be able to do
```
register cleanUp api.workspaces.delete(projectName, cloneWorkspaceName)
```